src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -104,7 +103,13 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
           </div>
         )}
         {hasUsers && (
-          <div className="PosterPreview__visiting">{userCountText}</div>
+          <div className="PosterPreview__users">
+            <FontAwesomeIcon
+              className="PosterPreview__users-icon"
+              icon={faUserAlt}
+            />
+            {userCount}
+          </div>
```

Comment:
FWIW, here is how it looks like  as of 212773beda38578f3cbdb06acac8339ee8898359 (I with there were some "release tags" in this repo)

![image](https://user-images.githubusercontent.com/39889/122087671-f7d9bb80-cdd2-11eb-94e0-4f0c83bcbab4.png)

Reply:
Yeah this is because of the poster Id being rendered. It's not in the design of the issue and I'm not sure how to proceed. When I get to it, I will ask Sofi about it.

Reply:
This is now aligned to be visually correct and not outside the container.